geometry2/12605ab/12605ab.bug

```diff
@@ -63,4 +63,4 @@ fault-codes:
   - PROGRAMMING:CONTROL-FLOW
 failure-codes:
   - SOFTWARE:LIVENESS
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:LOSS-OF-FUNCTIONALITY
```

Comment:
following our discussion in the meeting, we agreed that SYSTEM:UNRESPONSIVE also applies here